irdnatest.md

```diff
@@ -0,0 +1,5 @@
+# sine (rs)
```

Comment:
no need to put the #sine(rs) because we want all our profile to look similar

Reply:
Ok